nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
@danielmarbach - the only thing that's confusing is the context of "latest container adapter packages"
What's considered latest vs latest 4 months from now. Can we add a version specific language? Can we anchor it to the version of the NServiceBus core version that the containers use that exhibit this behavior?